index.html

```diff
@@ -26,41 +33,320 @@
         </a>
       </nav>
     </header>
-    <section>
-      <h1 id="first-section">I’m Tacettin Glad to see you!</h1>
-      <p id="second-section">
+    <section id="front-section">
+      <h1 id="first-subsection">I’m Tacettin Glad to see you!</h1>
+      <p id="second-subsection">
```

Comment:
- [ ] [OPTIONAL] Refrain from using `id`s too often, it is better to use `class`es instead. As classes are made so that code is re-usable, it makes more sense to use rather than using `id`s.

Reply:
You can be sure that I will pay attention to this issue in future studies.